{"name": "test_right_way", "status": "skipped", "statusDetails": {"message": "XFAIL Wait for fix bug\n\nselenium.common.exceptions.TimeoutException: Message:", "trace": "tests\\test_shop.py:90: in test_right_way\n    WebDriverWait(browser, timeout=10, poll_frequency=1).until(\n        browser    = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b66af65de3f8159adf9ad61b523c4024\")>\n        cart_is_visible = 'block'\n        product    = <selenium.webdriver.remote.webelement.WebElement (session=\"b66af65de3f8159adf9ad61b523c4024\", element=\"27734a50-9b60-4e8d-8987-7ab2f3033175\")>\n..\\..\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:90: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message:\n        end_time   = 28467.0\n        message    = ''\n        method     = <function url_to_be.<locals>._predicate at 0x0000025F2805DEA0>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"b66af65de3f8159adf9ad61b523c4024\")>\n        stacktrace = None\n        value      = False"}, "description": "\n    Test case TC-4\n    ", "attachments": [{"name": "stderr", "source": "479588f4-39bc-45aa-84a5-09c36b986390-attachment.txt", "type": "text/plain"}], "start": 1666802420094, "stop": 1666802463539, "uuid": "af84091e-58ab-43e5-b3d5-01dfbe1e021c", "historyId": "832350b147fac251f172289a626c5cba", "testCaseId": "7b8323124234f69a8fde1e3de50f452e", "fullName": "tests.test_shop#test_right_way", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(reason='Wait for fix bug')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_shop"}, {"name": "host", "value": "DESKTOP-NDLNMNH"}, {"name": "thread", "value": "18044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_shop"}]}